roles/eon-common/templates/eon.conf.j2

```diff
@@ -29,13 +29,15 @@ rabbit_hosts = "{{ rabbit_hosts }}"
 rabbit_use_ssl = "{{ rabbit_use_ssl }}"
 
 [keystone_authtoken]
-admin_tenant_name = "{{ eon_admin_tenant_name }}"
-admin_password = "{{ eon_admin_password }}"
-admin_user = "{{ eon_admin_user }}"
-auth_uri = "{{ keystone_auth_uri }}"
-identity_uri = "{{ keystone_identity_uri }}"
-endpoint_type = "{{ eon_endpoint_type }}"
-ca_certificates_file = "{{ eon_ca_certificates_file }}"
+auth_type = v3password
+auth_url = {{ keystone_auth_uri }}/v3
+auth_uri = {{ keystone_auth_uri }}/v3
+project_domain_name = Default
+project_name = {{ eon_admin_tenant_name }}
+user_domain_name = Default
+username = {{ eon_admin_user }}
```

Comment:
Have you tried running eon service with this change?   I see that the parameter names are being changed here.  These parameters are being referenced in the hpe-helion-os/hpe-eon module:  https://github.com/hpe-helion-os/hpe-eon/blob/0121bc128c0c69bbf0c73613ef7eae4715c54a99/eon/common/context.py

Reply:
Were you referring the function 'get_admin_context'? We don't have that in HOS5.0 code. It seems like there's a gap between two repos.

Reply:
@kwu83tw  I am assuming you are using both eon and eon-ansible repos from github.com/hpe-helion-os.    If so,  the changes that is added into the eon-ansible is not consistent with the eon module.